src/main/java/nl/tudelft/contextproject/model/entities/Carrot.java

```diff
@@ -43,13 +33,23 @@ public void setSpatial(Spatial spatial) {
 	}
 
 	@Override
-	public void update(float tpf) { }
+	public void update(float tpf) { 
```

Comment:
Please change to an else if, starting with the lowest health, so
`if (health <= 1) {
...
} else if (health <= 3) {
...
}
`

This would make it so that models do not get loaded for no reason.